docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
@FireFart currently you can use `docker-compose run --rm ms ./msfvenom` to invoke `msfvenom`. Is that sufficient?